app-config-react-native-transformer/README.md

```diff
@@ -0,0 +1,48 @@
+# React Native App Config Transformer
+React Native Metro transformer that loads your `app-config` values into bundles statically.
+
+## Installation
+
+### 1. Install `@lcdev/app-config`
+
+```shell
+yarn add @lcdev/app-config
+```
+
+Or, if you use NPM.
+
+```shell
+npm i @lcdev/app-config
+```
+
+### 2. Install `@lcdev/react-native-app-config-transformer`
+
+```shell
+yarn add -D @lcdev/react-native-app-config-transformer
+```
+
+Or, if you use NPM.
+
+```shell
+npm i -D @lcdev/react-native-app-config-transformer
+```
+
+### 3. Configure React Native bundler (Metro)
+Merge the contents of your project's `metro.config.js` file with this config (or create the file if it does not exist already).
+
+`metro.config.js:`
+
+```javascript
+module.exports = (async () => {
+  return {
+    // Force cache reset in order to load app-config changes
+    resetCache: true,
+    transformer: {
+      babelTransformerPath: require.resolve('@lcdev/react-native-app-config-transformer'),
```

Comment:
Do I understand it correctly to say that technically, this is a babel transformer? Could it be generalized in the future?

Reply:
Unfortunately, no. The "babel transformer" referred to here is a higher level function created by Metro.

As I understand it, `tranform()` behaves more like a loader (called for every module in the app, you decide how to parse it). The option `babelTransformerPath` is perhaps a poor naming choice by Metro - its primary use is to transform via babel, but it is not restricted to that. 

See https://github.com/facebook/metro/blob/df8fd4cc18ead6ef092d22f0c65c264629e53a33/packages/metro-react-native-babel-transformer/src/index.js#L174

Reply:
Good to know, thanks!